src/app/(home)/page.tsx

```
@@ -0,0 +1,67 @@
+		<HydrateClient>
+			<div className="min-h-[80vh] flex items-center justify-center overflow-hidden no-visible-scrollbar px-6 md:px-0">
+				<main className="flex flex-col gap-4 row-start-2 items-center justify-center">
+					<div className="flex flex-col gap-1">
+						<h3 className="font-bold text-4xl text-black dark:text-white text-center">
+							GlucoTrack
+						</h3>
+						<p className="text-center break-words text-sm md:text-base">
+							Official demo to showcase{" "}
+							<a
+								href="https://better-auth.com"
+								target="_blank"
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)